src/components/Upload.tsx

```diff
@@ -1,13 +1,13 @@
 import { useEffect } from 'react';
-import { useDataTable } from '~/hooks/useDataTable';
+import { useDataTable } from '../hooks/useDataTable';
```

Comment:
why is that change needed? `~` should point to repo root, that seems better to me